client/components/Video.jsx

```diff
@@ -54,30 +68,35 @@ class Video extends React.Component {
     }
   }
 
+  syncVideos(video, otherTime) {
+    if (Math.floor(video.currentTime) > Math.floor(otherTime) + 0.5 ||
+        Math.floor(video.currentTime) < Math.floor(otherTime) - 0.5) {
+      video.currentTime = otherTime;
+    }
+  }
+
   emitPlayAndListenForPause(eventOrVideo) {
     const video = eventOrVideo.target ? eventOrVideo.target : eventOrVideo;
     this.props.socket.emit('play', video.currentTime);
     this.props.socket.on('pause', (otherTime) => {
```

Comment:
Following up on our discussion -- would move this event listener to the componentDidMount function. This could be bound in an initListeners() method that would bind all listeners (onplay, onpause, ongoback)